proto/storage/notifier.proto

```diff
@@ -161,6 +161,9 @@ message MicrosoftSentinel {
     string private_key = 2; // @gotags: scrub:"always"
   }
   ClientCertAuthConfig client_cert_auth_config = 7;
+
+  // Enables authentication with short-lived tokens using Azure managed identities or Azure workload identities.
+  bool wif_enabled = 8;
```

Comment:
Any reason it can't always be true? Any risk for having this enabled even if not used?

Reply:
It could be done - some arguments against it:

* having an explicit toggle makes it feel less like "magic".
* a toggle is consistent with how it already works for other integrations. Also for image integrations we probably don't want to always enable WIF because of the overhead in combination with how registry matching works (today).

Reply:
I see, thanks. Let's add that note to the comment so users can learn this info as well.